packages/gitbook/src/lib/references.tsx

```diff
@@ -272,6 +277,27 @@ export async function resolveContentRef(
             };
         }
 
+        case 'openapi': {
+            if (!siteContext) {
+                return null;
+            }
+            const { organizationId } = siteContext;
+            const [openAPISpecVersion, openAPISpecVersionContent] = await Promise.all([
+                getLatestOpenAPISpecVersion(organizationId, contentRef.spec),
+                getLatestOpenAPISpecVersionContent(organizationId, contentRef.spec),
```

Comment:
This is inefficient, we should have optimized the content API to also return the URL

Reply:
Yep, it's an optimization we should do.

Reply:
[RND-6305: Add URL to getLatestOpenAPISpecVersionContent](https://linear.app/gitbook-x/issue/RND-6305/add-url-to-getlatestopenapispecversioncontent)